utils/trivialize-rules/explode-regexp.js

```diff
@@ -4,8 +4,8 @@ const { parse } = require('regulex');
 
 class UnsupportedRegExp extends Error {}
 
-function explodeRegExp(re, callback) {
-  (function buildUrls(str, items) {
+function explodeRegExp (re, callback) {
```

Comment:
Hmm, this file was previously formatted with prettier. Did it add these whitespaces or did you manually? Looks quite odd.

Reply:
May I know which prettier options did you used? I can revert the changes in a37df21

Reply:
I think just single-quotes plus semicolons.

Reply:
It seems to me that prettier will create some subtle changes to `trivialize-rules.js`, e.g. 

```diff
+            let scReSrc = `\n([\t ]*)<securecookie\\s*host=\\s*"${escapeStringRegexp(
+              securecookie.host
+            )}"(\\s*)name=\\s*"${escapeStringRegexp(
+              securecookie.name
+            )}"\\s*?/>[\t ]*\n`;
``` 

So I guess it is better not to run prettier on it. Would you mind if I leave the file as-is in e8f27ee?